cmd/lakectl/cmd/common_helpers.go

```diff
@@ -32,6 +32,11 @@ var ErrInvalidValueInList = errors.New("empty string in list")
 
 var accessKeyRegexp = regexp.MustCompile(`^AKIA[I|J][A-Z0-9]{14}Q$`)
 
+// const values used by lakectl commands
+const (
```

Comment:
general note - not actionable to this pr - but it looks like we have a lot of 'const' blocks here